docs/website/blog/2023-12-01-dlt-kestra-demo.md

```diff
@@ -0,0 +1,170 @@
+This implementation doesn't handle email attachments, but if you need to analyze, for instance, invoice PDFs from your inbox, you can read about how to automate this with `dlt` [here](https://dlthub.com/docs/blog/dlt-ops-startups).
+:::
+
+## The AI part
+
+>💡 In this day and age, how can we not incorporate AI into everything? 😆
+
+But seriously, if you're familiar with OpenAI, it's a matter of an API call to the chat completion endpoint. What simplifies it even further is `Kestra`’s [OpenAI plugin](https://kestra.io/plugins/plugin-openai).
+
+In my [subflow](https://github.com/dlt-hub/dlt-kestra-demo/blob/main/subflow.yml), I used it to obtain both the summary and sentiment analysis of each email body. Here's a glimpse of how it's implemented:
+
+```yaml
+- id: get_summary
+  type: io.kestra.plugin.openai.ChatCompletion
+  apiKey: "{{ secret('OPENAI_API') }}"
+  model: gpt-3.5-turbo
+  prompt: "Summarize the email content in one sentence with less than 30 words: {{inputs.data[0]['body']}}"
+  messages: [{"role": "system", "content": "You are a tool that summarizes emails."}]
+```
+
+:::info
+`Kestra` also includes Slack, as well as BigQuery plugins, which I used in my flows. Additionally, there is a wide variety of [other plugins](https://kestra.io/plugins) available.
+:::
+
+## The automation part
+
+>💡 `Kestra` triggers are the ideal solution!
+
+I’ve used a [schedule trigger](https://kestra.io/docs/developer-guide/triggers) that allows you to execute your flow on a regular cadence e.g. using a CRON expression:
+
+```yaml
+triggers:
+  - id: schedule
+    type: io.kestra.core.models.triggers.types.Schedule
+    cron: "0 9-18 * * 1-5"
+```
+
+This configuration ensures that your flows are executed hourly on workdays from 9 AM to 6 PM.
+
+---
+
+# THE OUTCOME
+
+A Slack assistant that delivers crisp inbox insights right at your fingertips:
+
+![slack.png](https://storage.googleapis.com/dlt-blog-images/dlt_kestra_slack.png)
+
+And a well-organized table in BigQuery, ready for you to dive into a more complex analysis:
+
+![bigquery_test.png](https://storage.googleapis.com/dlt-blog-images/dlt_kestra_bigquery_test.png)
+
+In essence, using **`Kestra`** and **`dlt`** offers a trio of advantages for refining email analysis and data workflows:
+
+1. **Efficient Automation**: Kestra effortlessly orchestrates intricate workflows, integrating smoothly with tools like **`dlt`**, OpenAI, and BigQuery. This streamlined process reduces manual intervention while eliminating errors, and freeing up more time for you.
+2. **User-Friendly and Versatile**: Both **`Kestra`** and **`dlt`** are crafted for ease of use, accommodating a range of skill levels. Their adaptability extends to various use cases.
+3. **Seamless Scaling**: **`Kestra`**, powered by Kafka and Elasticsearch, adeptly manages large-scale data and complex workflows. Coupled with **`dlt`**'s solid data integration capabilities, it ensures a stable and reliable solution for diverse requirements.
```

Comment:
this list looks like ChatGPT generated, hi-hi.
at least I wouldn't capitalize every word:

1. Efficient automation: ...
2. User-friendly and versatile: ...

Reply:
This is a writing convention, though - but will do